pkg/cortex/modules.go

```diff
@@ -177,6 +184,15 @@ func (m *moduleName) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return m.Set(s)
 }
 
+func (m moduleName) IsJob() bool {
+	switch m {
+	case Flusher:
+		return true
+	}
+
+	return false
+}
+
```

Comment:
Is this method used anywhere? If not, should it be removed?

Reply:
Dead code from old iterations. Will remove it.